stake-pool/program/tests/huge_pool.rs

```diff
@@ -26,13 +26,13 @@ use {
         find_withdraw_authority_program_address, id,
         instruction::{self, PreferredValidatorType},
         stake_program,
-        state::{AccountType, StakePool, StakeStatus, ValidatorList, ValidatorStakeInfo},
+        state::{AccountType, Fee, StakePool, StakeStatus, ValidatorList, ValidatorStakeInfo},
         MAX_VALIDATORS_TO_UPDATE, MINIMUM_ACTIVE_STAKE,
     },
     spl_token::state::{Account as SplAccount, AccountState as SplAccountState, Mint},
 };
 
-const HUGE_POOL_SIZE: u32 = 4_000;
+const HUGE_POOL_SIZE: u32 = 3950;
```

Comment:
Ah man, what a shame, this makes me sad.  With no withdrawal fee, we can't get to 4k? That seems weird, unless deserialization of the new fields is that heavy.  Which tests fail at 4k?

Reply:
The test which fails is upgrade for big_pools. Specifically, it is update pool values which fails. I tested that master fails with pool size 4100, so we didn't regress **that** much... 🙂

Reply:
hehe clever me noticed that `apply_fee` wasn't inlined, we're back at `4_000` now  :muscle: